template/.readme/partials/borrowed/documentation.md.j2.j2

```diff
@@ -0,0 +1,5 @@
+## Documentation
+
+The documentation for this operator can be found [here](https://docs.stackable.tech/{{operator_name}}/stable/index.html).
```

Comment:
Should we point to nightly? This will better reflect the current docs when landing on "main"?
```suggestion
The documentation for this operator can be found [here](https://docs.stackable.tech/{{operator_name}}/nightly/index.html).
```

Reply:
Added a change to reference both versions explicitly.